tycho-plugins/repository-utils/src/test/java/org/jboss/tools/tycho/sitegenerator/SystemPropertiesTest.java

```diff
@@ -10,14 +10,10 @@
 public class SystemPropertiesTest {
 
 	@Test
-	public void testHostName() {
+	public void testHostName() throws UnknownHostException {
 		java.net.InetAddress localMachine;
-		try {
-			localMachine = java.net.InetAddress.getLocalHost();
-			System.setProperty("HOSTNAME",localMachine.getHostName());
-		} catch (UnknownHostException e) {
-			e.printStackTrace();
-		}
+		localMachine = java.net.InetAddress.getLocalHost();
+		System.setProperty("HOSTNAME",localMachine.getHostName());
```

Comment:
It's not testing much, as discussed today. It's testing that java.net.InetAddress.getLocalHost() returns a non-null value. 

So... it could be better, but it's out of scope for this PR.